### DISCUSSION TITLE
Generate mesh files

### DISCUSSION THREAD

**Discussion by zkemp40** [gwlucastrig/Tinfour#97] [General]
Is it possible to generate and meshed .obj or .off file from the tool? I have a .las file that I'd like to convert.

**Comment by gwlucastrig** [comment]
I've never implemented a capability to do that... In fact, I'm unfamiliar with the data formats you mention.  If you can provide a bit more detail, I can estimate a level of effort for an implementation.

**Comment by zkemp40** [comment]
Thanks for the reply. A OBJ file format stores information about 3D models. It was created by [Wavefront Technologies for its Advanced Visualizer application to store geometric objects composed of lines, polygons, and freeform curves and surfaces. What I'm looking to do is create a TIN out of the point cloud data that I have.

**Comment by zkemp40** [comment]
Here's a good site on OBJ files. https://all3dp.com/1/obj-file-format-3d-printing-cad/
